src/main/java/edu/greenblitz/robotName/commands/arm/GoToAndScoreToAmp.java

```diff
@@ -0,0 +1,14 @@
+package edu.greenblitz.robotName.commands.arm;
+
+import edu.greenblitz.robotName.commands.swerve.MoveToAmp;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+
+public class GoToAndScoreToAmp extends ParallelCommandGroup {
+
+    public GoToAndScoreToAmp() {
+        super(
+                new ScoreToAmp(),
+                new MoveToAmp()
```

Comment:
Parallel of scoreToAmp and moveToAmp? Does score to amp wait for the right chassis pose? Or will this not work?

Reply:
Again this is parallel so the note can be scored before the chassis is in scoring position